paasta_tools/graceful_app_drain.py

```diff
@@ -73,7 +73,8 @@ def main():
         (old_app_live_happy_tasks,
          old_app_live_unhappy_tasks,
          old_app_draining_tasks,
-         ) = get_old_happy_unhappy_draining_tasks(
+         old_app_at_risk_tasks,
```

Comment:
is this right? I think the diff is saying that both line 75 and 76 are now the same?